jest.config.base.js

```diff
@@ -47,5 +47,6 @@ module.exports = {
         '^bcrypto/lib/(.*)$': 'bcrypto/lib/$1-browser',
         // Enforce usage of CommonJS version of uuid because ESM version is not working in Jest
         '^uuid$': require.resolve('uuid'), // https://stackoverflow.com/questions/73203367/jest-syntaxerror-unexpected-token-export-with-uuid-library
+        '^uint8array-tools$': require.resolve('uint8array-tools'), // same case as in uuid
```

Comment:
This lib has been added as sub-dependency of `varuint-bitcoin`.
I checked [the code](https://github.com/bitcoinjs/uint8array-tools) that it contains nothing sus.
But like `uuid`, it has misconfigured CJS vs MJS exports. This has to be done so that jest can resolve the import 

:information_source: :bulb: FIY, this is only necessary for test suites using `jsdom` env. The `utxo-lib` test suite has `node` env, and it can handle the import fine, so its test passes even without this fix, but it would fail in _other packages_, where `utxo-lib` package is indirectly used